stitch/src/main/java/com/mongodb/stitch/android/auth/AvailableAuthProviders.java

```diff
@@ -16,21 +21,25 @@
     private final GoogleAuthProviderInfo _google;
     private final AnonymousAuthProviderInfo _anon;
     private final EmailPasswordAuthProviderInfo _emailPass;
+    @NonNull
```

Comment:
I don't think you need this if the getter is annotated as @Nullable.

Reply:
The getter should've been marked as @nonnull. That's my mistake.